spec/integration/api_resources_controller_spec.rb

```diff
@@ -21,10 +21,10 @@
             type: :object, properties: {
               'id': { type: :integer },
               'name': { type: :string },
-              'url': { type: :string, nullable: true }
+              'url': { type: :string, nullable: true },
+              'average_evaluation': { type: :number, nullable: true }
```

Comment:
The API returns average_evaluation as string, but the in the schema is defined as number